pages/upgrade-guide/4.0.md

```diff
@@ -139,27 +181,33 @@ jQuery 4.0 switches strategies when it comes to setting CSS properties that requ
 
 Instead, jQuery will now only add `px` to a limited set of properties that are known to require it. This change may affect existing code that relies on the old behavior.
 
+#### Breaking change: Remove opacity CSS hook
+
+`.css( "opacity" )` will now return an empty string for detached elements in standard-compliant browsers and "1" in IE. That behavior is shared by most other CSS properties, but may affect existing code relying on a return value of "1".
+
 ### Data
 
 #### Important fix: `Object.prototype` pollution
 
-jQuery 4.0 includes a fix to ensure event and data keys matching `Object.prototype` properties are supported without overriding the native properties. This change is not expected to affect existing code unless it specifically used native property names as event or data keys, which may have caused unexpected behavior anyway.
+jQuery 4.0 includes a fix to ensure event and data keys matching `Object.prototype` properties are supported without overriding the native properties. This change is not expected to affect existing code.
 
 ### Deferred
 
-#### Breaking change: Removed jQuery.Deferred.getStackHook
+#### Breaking change: Removed `jQuery.Deferred.getStackHook`
 
-This has been renamed since 3.7.0 to jQuery.Deferred.getErrorHook. It is used to pass the original error to jQuery.Deferred.exceptionHook. Previously, we recommended passing the stack but that didn't play well with source maps.
+This has been renamed since 3.7.0 to `jQuery.Deferred.getErrorHook`. It is used to pass the original error to `jQuery.Deferred.exceptionHook`. Previously, we recommended passing the stack but that didn't play well with source maps. `jQuery.Deferred.exceptionHook` can be used to debug asynchronous errors.
+
+TODO: link to documentation
```

Comment:
I have a docs PR at https://github.com/jquery/api.jquery.com/pull/1277, if we can get it landed, we'll be able to add a link here.

Reply:
These are now live:
* https://api.jquery.com/jQuery.Deferred.getStackHook/
* https://api.jquery.com/jQuery.Deferred.getErrorHook/